### ISSUE TITLE
Appium Desktop screenshot is not working

### ISSUE THREAD

**Issue by paulinx** [appium/appium-desktop#1911]
## Appium or Appium Desktop?

You are reporting an issue at the Appium Desktop repository. Appium Desktop is a wrapper around Appium. If you are having trouble running tests, it is much more likely that the problem you are encountering is not a problem with Appium Desktop but with Appium. For that reason we require you to have tried your testcase with Appium CLI before reporting issues. Check the checkbox below to confirm that you have proven your issue does _not_ reproduce on Appium itself:

- [V] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem

I don't know why My Appium Desktop can not use when I update IOS to 15 and Xcode to 13.

![A4AC3FD4-0AF3-41EF-A2F1-979994D2743B](https://user-images.githubusercontent.com/16395127/134468464-43edb915-5b9a-4813-a9d7-5a96b5092919.jpeg)

![F988A1B5-AA40-473D-BC94-BF7942D26D4A](https://user-images.githubusercontent.com/16395127/134468520-0befe99a-5336-4d3b-aa5b-162bf125bb0a.jpeg)

And I use JAVA to run screenshot. It is wrong,too.
![7CC79885-F9D0-413D-99A2-F47E2ACC7C96](https://user-images.githubusercontent.com/16395127/134468784-1ab46f9b-875e-435a-bf9a-d153a5aa0abc.jpeg)

I found my real device has this information "Automation Running" 
![S__382214150](https://user-images.githubusercontent.com/16395127/134469399-cfa5a2cf-16ac-47f1-a5c5-bddcd9a18255.jpg)

How can I work screenshot?

Thanks.

## Environment
IOS 15.0
Xcode 13.0 

- I am running Appium Desktop version _<insert version here>_.
App Version: 1.21.0
Electron: 7.2.4
Node.js: 12.8.1
- I am on (pick one):
    - [V] Mac
    - [ ] Windows
    - [ ] Linux

**Comment by wswebcreation** [comment]
Hi @paulinx 

iOS 15 is new and is only supported by Appium 1.22.0. You can try to install a local version of Appium 1.22.0 and connect to that one. That should work. Of you can use the new https://github.com/appium/appium-inspector

**Comment by KazuCocoa** [comment]
Yes. Please use Appium 1.22.0 for iOS 15.

**Comment by paulinx** [comment]
@KazuCocoa  Sorry I used Appium Server GUI (1.22.0)and Appium Inspector(2021.9.1)
But I still wrong. QQ

![9E5A1260-821B-4658-B9E2-59AA4202DD20](https://user-images.githubusercontent.com/16395127/134833195-cec98cf6-75f3-4858-b50a-e2797c7fbbad.jpeg)

![9C7D68A3-8AD1-484A-81F9-4B469B0C0266](https://user-images.githubusercontent.com/16395127/134833253-9c5a73eb-bd88-491a-927d-9b3eb256bc5e.jpeg)

Or Should I set up something in my real devices?

Thanks you so much.
